### DISCUSSION TITLE
espeak on getty

### DISCUSSION THREAD

**Discussion by ranomier** [espeak-ng/espeak-ng#1938] [Q&A]
I am working with a blind person and i would like to reduce complexity.

A lot of display manager (login manager) don't support text to speech or are weird with it.

Would it be possible to run espeak-ng after boot up on a getty login?

**Comment by jaacoppi** [comment]
Do you mean you'd want to use espeak-ng as a TTS on the console/terminal instead of an X/Wayland session? Sure, just get a screen reader that reads the terminal and passes it to espeak-ng. Speech Dispatcher should work.
